### PR TITLE
Fixed broken links in computer_science/CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by mdfaizanahmed786**
All broken links have been fixed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## New contributor guide
 
-To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+To get an overview of the project, read the [README](https://github.com/mdfaizanahmed786/computer_science/blob/contributingmd-links-fix/readme.md). Here are some resources to help you get started with open source contributions:
```
Comment (shhossain) [inline CONTRIBUTING.md:3]: Change this link to "/readme.md" .

**Review by shhossain — CHANGES_REQUESTED** [review]
Change link

File: CONTRIBUTING.md (hunk 1 above)
Comment (mdfaizanahmed786) [reply CONTRIBUTING.md:3]: I'm sorry, I've actually added my link.

Comment (shhossain) [reply CONTRIBUTING.md:3]: No problem. Just change it and push it.

**Review by shhossain — CHANGES_REQUESTED** [review]
Just change to /readme.md

**Comment by mdfaizanahmed786** [comment]
Hi there, I updated the link.

**Comment by shhossain** [comment]
Merging...

**Comment by mdfaizanahmed786** [comment]
I hope you've liked my work. This was my first open-source contribution. I know to understand it completely I've made some mistakes but I managed to contribute to your repository.

**Comment by shhossain** [comment]
Congratulations 🎊.

**Comment by mdfaizanahmed786** [comment]
Thanks! ❤️

**Comment by mdfaizanahmed786** [comment]
> Congratulations 🎊.

> Hey there, can you please mark by PR as hacktober-fest? It would be a great help.

**Comment by shhossain** [comment]
@mdfaizanahmed786 To get "hactoberfest-accepted" label you need to add info and it must be unique. minimum ➕200 addition.

**Comment by mdfaizanahmed786** [comment]
> @mdfaizanahmed786 To get "hactoberfest-accepted" label you need to add info and it must be unique. minimum ➕200 addition.

> If you don't mind, can you describe in detail what info I need to add? Thank you for you time!

**Comment by shhossain** [comment]
@mdfaizanahmed786 See this [issue](https://github.com/shhossain/computer_science/issues/1)

**Comment by mdfaizanahmed786** [comment]
> @mdfaizanahmed786 See this [issue](https://github.com/shhossain/computer_science/issues/1)

> Got it! Thanks!

**Pull request merged**